```
Error running mint automation
+ mkdir -p 10077-8f92792/gopath/src/github.com/minio
+ git -C 10077-8f92792/gopath/src/github.com/minio clone --quiet https://github.com/minio/minio.git
+ git -C 10077-8f92792/gopath/src/github.com/minio/minio remote add minio https://github.com/minio/minio.git
+ git -C 10077-8f92792/gopath/src/github.com/minio/minio fetch --quiet minio pull/10077/head:pr10077
+ git -C 10077-8f92792/gopath/src/github.com/minio/minio checkout --quiet pr10077
+ GO111MODULE=on
+ GOPATH=/home/mint-auto/mint-auto/10077-8f92792/gopath
+ make -C 10077-8f92792/gopath/src/github.com/minio/minio --quiet
go: downloading github.com/minio/cli v1.22.0
go: downloading github.com/dustin/go-humanize v1.0.0
go: downloading github.com/minio/minio-go/v7 v7.0.1
go: downloading google.golang.org/api v0.5.0
go: downloading github.com/shirou/gopsutil v2.20.3-0.20200314133625-53cec6b37e6a+incompatible
go: downloading github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: downloading golang.org/x/crypto v0.0.0-20200709230013-948cd5f35899
go: extracting github.com/dustin/go-humanize v1.0.0
go: downloading github.com/willf/bloom v2.0.3+incompatible
go: extracting github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: downloading github.com/Azure/azure-storage-blob-go v0.8.0
go: extracting github.com/minio/cli v1.22.0
go: extracting github.com/shirou/gopsutil v2.20.3-0.20200314133625-53cec6b37e6a+incompatible
go: extracting github.com/willf/bloom v2.0.3+incompatible
go: downloading github.com/minio/highwayhash v1.0.0
go: downloading github.com/gorilla/handlers v1.4.2
go: extracting github.com/minio/minio-go/v7 v7.0.1
go: extracting github.com/Azure/azure-storage-blob-go v0.8.0
go: extracting github.com/minio/highwayhash v1.0.0
go: extracting github.com/gorilla/handlers v1.4.2
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
go: downloading github.com/gorilla/rpc v1.2.0
go: extracting github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
go: extracting github.com/minio/sha256-simd v0.1.1
go: downloading github.com/beevik/ntp v0.2.0
go: downloading github.com/willf/bitset v1.1.10
go: downloading github.com/prometheus/client_golang v1.0.0
go: downloading github.com/djherbis/atime v1.0.0
go: downloading github.com/minio/sio v0.2.0
go: extracting github.com/gorilla/rpc v1.2.0
go: extracting github.com/beevik/ntp v0.2.0
go: downloading github.com/klauspost/compress v1.10.3
go: downloading golang.org/x/sys v0.0.0-20200625212154-ddb9806d33ae
go: extracting github.com/djherbis/atime v1.0.0
go: downloading github.com/colinmarc/hdfs/v2 v2.1.1
go: extracting github.com/minio/sio v0.2.0
go: downloading github.com/secure-io/sio-go v0.3.0
go: extracting github.com/prometheus/client_golang v1.0.0
go: downloading github.com/klauspost/readahead v1.3.1
go: extracting github.com/secure-io/sio-go v0.3.0
go: downloading github.com/klauspost/reedsolomon v1.9.9
go: extracting github.com/klauspost/readahead v1.3.1
go: downloading gopkg.in/jcmturner/gokrb5.v7 v7.3.0
go: extracting github.com/colinmarc/hdfs/v2 v2.1.1
go: extracting golang.org/x/sys v0.0.0-20200625212154-ddb9806d33ae
go: downloading github.com/dchest/siphash v1.2.1
go: downloading github.com/Azure/azure-pipeline-go v0.2.1
go: extracting github.com/willf/bitset v1.1.10
go: downloading github.com/spaolacci/murmur3 v1.1.0
go: extracting github.com/dchest/siphash v1.2.1
go: extracting github.com/Azure/azure-pipeline-go v0.2.1
go: downloading github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4
go: extracting github.com/spaolacci/murmur3 v1.1.0
go: extracting github.com/klauspost/reedsolomon v1.9.9
go: downloading github.com/mattn/go-ieproxy v0.0.1
go: extracting github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4
go: downloading github.com/elazarl/go-bindata-assetfs v1.0.0
go: extracting github.com/mattn/go-ieproxy v0.0.1
go: downloading github.com/klauspost/cpuid v1.3.1
go: extracting golang.org/x/crypto v0.0.0-20200709230013-948cd5f35899
go: extracting github.com/klauspost/cpuid v1.3.1
go: downloading cloud.google.com/go v0.39.0
go: downloading github.com/google/uuid v1.1.1
go: downloading github.com/json-iterator/go v1.1.10
go: extracting github.com/google/uuid v1.1.1
go: downloading github.com/cheggaaa/pb v1.0.28
go: extracting github.com/elazarl/go-bindata-assetfs v1.0.0
go: downloading github.com/cespare/xxhash/v2 v2.1.1
go: extracting github.com/cespare/xxhash/v2 v2.1.1
go: downloading github.com/beorn7/perks v1.0.0
go: extracting github.com/cheggaaa/pb v1.0.28
go: downloading github.com/miekg/dns v1.1.8
go: extracting github.com/beorn7/perks v1.0.0
go: downloading github.com/tinylib/msgp v1.1.2
go: extracting github.com/json-iterator/go v1.1.10
go: extracting github.com/miekg/dns v1.1.8
go: extracting github.com/tinylib/msgp v1.1.2
go: downloading go.etcd.io/etcd/v3 v3.3.0-rc.0.0.20200707003333-58bb8ae09f8e
go: extracting cloud.google.com/go v0.39.0
go: downloading github.com/prometheus/common v0.4.1
go: extracting github.com/prometheus/common v0.4.1
go: downloading gopkg.in/ini.v1 v1.57.0
go: extracting gopkg.in/ini.v1 v1.57.0
go: extracting go.etcd.io/etcd/v3 v3.3.0-rc.0.0.20200707003333-58bb8ae09f8e
go: downloading github.com/klauspost/pgzip v1.2.1
go: downloading google.golang.org/grpc v1.26.0
go: extracting github.com/klauspost/pgzip v1.2.1
go: extracting google.golang.org/grpc v1.26.0
go: downloading github.com/coredns/coredns v1.4.0
go: extracting github.com/coredns/coredns v1.4.0
go: downloading golang.org/x/net v0.0.0-20200707034311-ab3426394381
go: extracting golang.org/x/net v0.0.0-20200707034311-ab3426394381
go: downloading github.com/golang/protobuf v1.4.0
go: extracting github.com/golang/protobuf v1.4.0
go: downloading github.com/bcicen/jstream v0.0.0-20190220045926-16c1f8af81c2
go: extracting github.com/bcicen/jstream v0.0.0-20190220045926-16c1f8af81c2
go: downloading github.com/mattn/go-runewidth v0.0.2
go: extracting github.com/mattn/go-runewidth v0.0.2
go: extracting gopkg.in/jcmturner/gokrb5.v7 v7.3.0
go: extracting github.com/klauspost/compress v1.10.3
go: downloading github.com/minio/simdjson-go v0.1.5-0.20200303142138-b17fe061ea37
go: downloading github.com/modern-go/reflect2 v1.0.1
go: downloading go.uber.org/zap v1.14.1
go: extracting github.com/modern-go/reflect2 v1.0.1
go: extracting go.uber.org/zap v1.14.1
go: downloading go.uber.org/atomic v1.6.0
go: downloading github.com/montanaflynn/stats v0.5.0
go: downloading gopkg.in/ldap.v3 v3.0.3
go: downloading github.com/alecthomas/participle v0.2.1
go: downloading gopkg.in/jcmturner/goidentity.v3 v3.0.0
go: downloading google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
go: extracting go.uber.org/atomic v1.6.0
go: downloading github.com/fatih/color v1.7.0
go: extracting github.com/montanaflynn/stats v0.5.0
go: extracting gopkg.in/jcmturner/goidentity.v3 v3.0.0
go: extracting gopkg.in/ldap.v3 v3.0.3
go: downloading github.com/hashicorp/go-uuid v1.0.1
go: downloading google.golang.org/protobuf v1.22.0
go: extracting github.com/alecthomas/participle v0.2.1
go: extracting github.com/minio/simdjson-go v0.1.5-0.20200303142138-b17fe061ea37
go: downloading github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: extracting github.com/hashicorp/go-uuid v1.0.1
go: extracting github.com/fatih/color v1.7.0
go: extracting github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: downloading go.opencensus.io v0.21.0
go: extracting go.opencensus.io v0.21.0
go: extracting google.golang.org/protobuf v1.22.0
go: extracting google.golang.org/api v0.5.0
go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading github.com/rs/cors v1.7.0
go: downloading github.com/ncw/directio v1.0.5
go: extracting github.com/ncw/directio v1.0.5
go: extracting github.com/rs/cors v1.7.0
go: extracting github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading gopkg.in/yaml.v2 v2.2.8
go: downloading github.com/gogo/protobuf v1.3.1
go: downloading github.com/mattn/go-isatty v0.0.8
go: extracting gopkg.in/yaml.v2 v2.2.8
go: downloading github.com/hashicorp/vault/api v1.0.4
go: extracting github.com/mattn/go-isatty v0.0.8
go: extracting github.com/hashicorp/vault/api v1.0.4
go: downloading github.com/hashicorp/golang-lru v0.5.1
go: extracting github.com/hashicorp/golang-lru v0.5.1
go: downloading github.com/prometheus/procfs v0.0.8
go: extracting github.com/prometheus/procfs v0.0.8
go: downloading github.com/hashicorp/vault/sdk v0.1.13
go: extracting github.com/hashicorp/vault/sdk v0.1.13
go: downloading github.com/philhofer/fwd v1.0.0
go: extracting github.com/philhofer/fwd v1.0.0
go: downloading github.com/gomodule/redigo v2.0.0+incompatible
go: extracting github.com/gogo/protobuf v1.3.1
go: extracting github.com/gomodule/redigo v2.0.0+incompatible
go: downloading github.com/nats-io/stan.go v0.7.0
go: extracting github.com/nats-io/stan.go v0.7.0
go: downloading github.com/hashicorp/hcl v1.0.0
go: extracting google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
go: extracting github.com/hashicorp/hcl v1.0.0
go: downloading github.com/hashicorp/errwrap v1.0.0
go: downloading github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
go: downloading github.com/hashicorp/go-retryablehttp v0.5.4
go: downloading github.com/eclipse/paho.mqtt.golang v1.2.0
go: downloading github.com/rjeczalik/notify v0.9.2
go: downloading github.com/mattn/go-colorable v0.1.4
go: extracting github.com/hashicorp/errwrap v1.0.0
go: downloading github.com/nats-io/nats.go v1.10.0
go: extracting github.com/hashicorp/go-retryablehttp v0.5.4
go: extracting github.com/mattn/go-colorable v0.1.4
go: extracting github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
go: downloading github.com/streadway/amqp v0.0.0-20190404075320-75d898a42a94
go: downloading go.uber.org/multierr v1.5.0
go: downloading golang.org/x/oauth2 v0.0.0-20190226205417-e64efc72b421
go: extracting github.com/rjeczalik/notify v0.9.2
go: extracting github.com/eclipse/paho.mqtt.golang v1.2.0
go: downloading github.com/hashicorp/go-cleanhttp v0.5.1
go: extracting go.uber.org/multierr v1.5.0
go: downloading github.com/lib/pq v1.7.0
go: extracting github.com/nats-io/nats.go v1.10.0
go: downloading github.com/go-sql-driver/mysql v1.5.0
go: extracting github.com/streadway/amqp v0.0.0-20190404075320-75d898a42a94
go: extracting golang.org/x/oauth2 v0.0.0-20190226205417-e64efc72b421
go: downloading github.com/xdg/stringprep v1.0.0
go: downloading github.com/mitchellh/mapstructure v1.1.2
go: downloading github.com/coreos/go-semver v0.2.0
go: extracting github.com/hashicorp/go-cleanhttp v0.5.1
go: extracting github.com/lib/pq v1.7.0
go: downloading github.com/golang/snappy v0.0.1
go: extracting github.com/coreos/go-semver v0.2.0
go: extracting github.com/mitchellh/mapstructure v1.1.2
go: downloading github.com/hashicorp/go-multierror v1.0.0
go: extracting github.com/xdg/stringprep v1.0.0
go: downloading github.com/nats-io/nuid v1.0.1
go: downloading gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d
go: downloading gopkg.in/square/go-jose.v2 v2.3.1
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.1
go: extracting github.com/hashicorp/go-multierror v1.0.0
go: extracting github.com/nats-io/nuid v1.0.1
go: downloading github.com/jcmturner/gofork v0.0.0-20190328161633-dc7c13fece03
go: downloading golang.org/x/text v0.3.3
go: extracting gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d
go: extracting github.com/matttproud/golang_protobuf_extensions v1.0.1
go: extracting github.com/go-sql-driver/mysql v1.5.0
go: downloading github.com/tidwall/gjson v1.3.5
go: downloading github.com/pierrec/lz4 v2.4.0+incompatible
go: downloading github.com/fatih/structs v1.1.0
go: extracting github.com/jcmturner/gofork v0.0.0-20190328161633-dc7c13fece03
go: downloading gopkg.in/jcmturner/aescts.v1 v1.0.1
go: extracting gopkg.in/square/go-jose.v2 v2.3.1
go: downloading github.com/ryanuber/go-glob v1.0.0
go: extracting github.com/golang/snappy v0.0.1
go: downloading github.com/Shopify/sarama v1.24.1
go: extracting github.com/tidwall/gjson v1.3.5
go: downloading github.com/googleapis/gax-go/v2 v2.0.4
go: extracting gopkg.in/jcmturner/aescts.v1 v1.0.1
go: downloading golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
go: extracting github.com/fatih/structs v1.1.0
go: downloading github.com/hashicorp/go-sockaddr v1.0.2
go: extracting github.com/ryanuber/go-glob v1.0.0
go: downloading git.apache.org/thrift.git v0.13.0
go: downloading github.com/nats-io/jwt v0.3.2
go: downloading gopkg.in/olivere/elastic.v5 v5.0.80
go: downloading github.com/tidwall/sjson v1.0.4
go: extracting github.com/Shopify/sarama v1.24.1
go: extracting github.com/googleapis/gax-go/v2 v2.0.4
go: downloading github.com/mitchellh/go-homedir v1.1.0
go: extracting golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
go: downloading github.com/minio/md5-simd v1.1.0
go: downloading github.com/coreos/go-systemd/v22 v22.0.0
go: extracting github.com/hashicorp/go-sockaddr v1.0.2
go: downloading github.com/nats-io/nkeys v0.1.4
go: extracting github.com/tidwall/sjson v1.0.4
go: extracting github.com/nats-io/jwt v0.3.2
go: downloading github.com/gorilla/mux v1.7.5-0.20200711200521-98cb6bf42e08
go: downloading github.com/tidwall/match v1.0.1
go: extracting github.com/mitchellh/go-homedir v1.1.0
go: downloading github.com/pkg/errors v0.8.1
go: extracting github.com/coreos/go-systemd/v22 v22.0.0
go: extracting github.com/minio/md5-simd v1.1.0
go: downloading github.com/nsqio/go-nsq v1.0.7
go: extracting github.com/nats-io/nkeys v0.1.4
go: downloading github.com/hashicorp/go-rootcerts v1.0.1
go: extracting gopkg.in/olivere/elastic.v5 v5.0.80
go: downloading github.com/mailru/easyjson v0.0.0-20180730094502-03f2033d19d5
go: extracting github.com/tidwall/match v1.0.1
go: extracting github.com/gorilla/mux v1.7.5-0.20200711200521-98cb6bf42e08
go: downloading github.com/eapache/go-resiliency v1.1.0
go: extracting github.com/pkg/errors v0.8.1
go: downloading github.com/davecgh/go-spew v1.1.1
go: extracting github.com/nsqio/go-nsq v1.0.7
go: extracting github.com/hashicorp/go-rootcerts v1.0.1
go: downloading gopkg.in/jcmturner/dnsutils.v1 v1.0.1
go: extracting github.com/eapache/go-resiliency v1.1.0
go: downloading github.com/tidwall/pretty v1.0.0
go: extracting github.com/mailru/easyjson v0.0.0-20180730094502-03f2033d19d5
go: downloading github.com/rcrowley/go-metrics v0.0.0-20181016184325-3113b8401b8a
go: downloading gopkg.in/jcmturner/rpc.v1 v1.1.0
go: extracting github.com/davecgh/go-spew v1.1.1
go: extracting gopkg.in/jcmturner/dnsutils.v1 v1.0.1
go: downloading github.com/eapache/queue v1.1.0
go: extracting gopkg.in/jcmturner/rpc.v1 v1.1.0
go: downloading github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: extracting github.com/rcrowley/go-metrics v0.0.0-20181016184325-3113b8401b8a
go: extracting github.com/tidwall/pretty v1.0.0
go: extracting github.com/eapache/queue v1.1.0
go: extracting github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: extracting git.apache.org/thrift.git v0.13.0
go: extracting golang.org/x/text v0.3.3
go: extracting github.com/pierrec/lz4 v2.4.0+incompatible
go: finding google.golang.org/api v0.5.0
go: finding github.com/colinmarc/hdfs/v2 v2.1.1
go: finding gopkg.in/jcmturner/gokrb5.v7 v7.3.0
go: finding github.com/minio/minio-go/v7 v7.0.1
go: finding github.com/Azure/azure-pipeline-go v0.2.1
go: finding github.com/minio/cli v1.22.0
go: finding github.com/dustin/go-humanize v1.0.0
go: finding cloud.google.com/go v0.39.0
go: finding github.com/minio/sha256-simd v0.1.1
go: finding github.com/json-iterator/go v1.1.10
go: finding github.com/Azure/azure-storage-blob-go v0.8.0
go: finding github.com/montanaflynn/stats v0.5.0
go: finding golang.org/x/oauth2 v0.0.0-20190226205417-e64efc72b421
go: finding github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: finding github.com/minio/highwayhash v1.0.0
go: finding github.com/gorilla/mux v1.7.5-0.20200711200521-98cb6bf42e08
go: finding github.com/jcmturner/gofork v0.0.0-20190328161633-dc7c13fece03
go: finding gopkg.in/jcmturner/dnsutils.v1 v1.0.1
go: finding github.com/mattn/go-ieproxy v0.0.1
go: finding github.com/hashicorp/go-uuid v1.0.1
go: finding golang.org/x/crypto v0.0.0-20200709230013-948cd5f35899
go: finding google.golang.org/grpc v1.26.0
go: finding github.com/fatih/color v1.7.0
go: finding google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
go: finding github.com/googleapis/gax-go/v2 v2.0.4
go: finding github.com/mattn/go-colorable v0.1.4
go: finding github.com/mitchellh/go-homedir v1.1.0
go: finding github.com/mattn/go-isatty v0.0.8
go: finding github.com/golang/protobuf v1.4.0
go: finding github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: finding github.com/secure-io/sio-go v0.3.0
go: finding gopkg.in/ini.v1 v1.57.0
go: finding go.opencensus.io v0.21.0
go: finding golang.org/x/sys v0.0.0-20200625212154-ddb9806d33ae
go: finding github.com/shirou/gopsutil v2.20.3-0.20200314133625-53cec6b37e6a+incompatible
go: finding golang.org/x/net v0.0.0-20200707034311-ab3426394381
go: finding github.com/ncw/directio v1.0.5
go: finding github.com/google/uuid v1.1.1
go: finding github.com/minio/md5-simd v1.1.0
go: finding github.com/rjeczalik/notify v0.9.2
go: finding google.golang.org/protobuf v1.22.0
go: finding gopkg.in/jcmturner/aescts.v1 v1.0.1
go: finding gopkg.in/jcmturner/goidentity.v3 v3.0.0
go: finding github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: finding github.com/hashicorp/golang-lru v0.5.1
go: finding golang.org/x/text v0.3.3
go: finding github.com/klauspost/cpuid v1.3.1
go: finding github.com/modern-go/reflect2 v1.0.1
go: finding gopkg.in/jcmturner/rpc.v1 v1.1.0
go: finding github.com/cespare/xxhash/v2 v2.1.1
go: finding github.com/cheggaaa/pb v1.0.28
go: finding github.com/mattn/go-runewidth v0.0.2
go: finding github.com/dchest/siphash v1.2.1
go: finding github.com/djherbis/atime v1.0.0
go: finding github.com/elazarl/go-bindata-assetfs v1.0.0
go: finding github.com/gorilla/handlers v1.4.2
go: finding github.com/gorilla/rpc v1.2.0
go: finding github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
go: finding github.com/klauspost/compress v1.10.3
go: finding github.com/klauspost/readahead v1.3.1
go: finding github.com/klauspost/reedsolomon v1.9.9
go: finding github.com/miekg/dns v1.1.8
go: finding go.etcd.io/etcd/v3 v3.3.0-rc.0.0.20200707003333-58bb8ae09f8e
go: finding github.com/gogo/protobuf v1.3.1
go: finding go.uber.org/zap v1.14.1
go: finding go.uber.org/atomic v1.6.0
go: finding go.uber.org/multierr v1.5.0
go: finding github.com/coreos/go-systemd/v22 v22.0.0
go: finding github.com/coreos/go-semver v0.2.0
go: finding github.com/coredns/coredns v1.4.0
go: finding gopkg.in/ldap.v3 v3.0.3
go: finding gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d
go: finding github.com/Shopify/sarama v1.24.1
go: finding github.com/davecgh/go-spew v1.1.1
go: finding github.com/eapache/go-resiliency v1.1.0
go: finding github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: finding github.com/golang/snappy v0.0.1
go: finding github.com/eapache/queue v1.1.0
go: finding github.com/pierrec/lz4 v2.4.0+incompatible
go: finding github.com/rcrowley/go-metrics v0.0.0-20181016184325-3113b8401b8a
go: finding github.com/eclipse/paho.mqtt.golang v1.2.0
go: finding github.com/go-sql-driver/mysql v1.5.0
go: finding github.com/gomodule/redigo v2.0.0+incompatible
go: finding github.com/lib/pq v1.7.0
go: finding github.com/nats-io/nats.go v1.10.0
go: finding github.com/nats-io/jwt v0.3.2
go: finding github.com/nats-io/nkeys v0.1.4
go: finding github.com/nats-io/nuid v1.0.1
go: finding github.com/nats-io/stan.go v0.7.0
go: finding github.com/nsqio/go-nsq v1.0.7
go: finding github.com/pkg/errors v0.8.1
go: finding github.com/streadway/amqp v0.0.0-20190404075320-75d898a42a94
go: finding github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
go: finding github.com/xdg/stringprep v1.0.0
go: finding gopkg.in/olivere/elastic.v5 v5.0.80
go: finding github.com/mailru/easyjson v0.0.0-20180730094502-03f2033d19d5
go: finding github.com/hashicorp/vault/api v1.0.4
go: finding github.com/hashicorp/errwrap v1.0.0
go: finding github.com/hashicorp/go-cleanhttp v0.5.1
go: finding github.com/hashicorp/go-multierror v1.0.0
go: finding github.com/hashicorp/go-retryablehttp v0.5.4
go: finding github.com/hashicorp/go-rootcerts v1.0.1
go: finding github.com/hashicorp/hcl v1.0.0
go: finding github.com/hashicorp/vault/sdk v0.1.13
go: finding github.com/hashicorp/go-sockaddr v1.0.2
go: finding github.com/ryanuber/go-glob v1.0.0
go: finding github.com/mitchellh/mapstructure v1.1.2
go: finding golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
go: finding gopkg.in/square/go-jose.v2 v2.3.1
go: finding github.com/minio/sio v0.2.0
go: finding github.com/beevik/ntp v0.2.0
go: finding github.com/fatih/structs v1.1.0
go: finding gopkg.in/yaml.v2 v2.2.8
go: finding github.com/klauspost/pgzip v1.2.1
go: finding github.com/bcicen/jstream v0.0.0-20190220045926-16c1f8af81c2
go: finding github.com/alecthomas/participle v0.2.1
go: finding github.com/minio/simdjson-go v0.1.5-0.20200303142138-b17fe061ea37
go: finding git.apache.org/thrift.git v0.13.0
go: finding github.com/tidwall/gjson v1.3.5
go: finding github.com/tidwall/match v1.0.1
go: finding github.com/tidwall/pretty v1.0.0
go: finding github.com/tidwall/sjson v1.0.4
go: finding github.com/prometheus/client_golang v1.0.0
go: finding github.com/beorn7/perks v1.0.0
go: finding github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4
go: finding github.com/prometheus/common v0.4.1
go: finding github.com/matttproud/golang_protobuf_extensions v1.0.1
go: finding github.com/prometheus/procfs v0.0.8
go: finding github.com/rs/cors v1.7.0
go: finding github.com/tinylib/msgp v1.1.2
go: finding github.com/philhofer/fwd v1.0.0
go: finding github.com/willf/bloom v2.0.3+incompatible
go: finding github.com/spaolacci/murmur3 v1.1.0
go: finding github.com/willf/bitset v1.1.10
# github.com/minio/minio/cmd/config/etcd/dns
cmd/config/etcd/dns/etcd_dns.go:293:2: cannot use args (type *CoreDNS) as type Store in return argument:
	*CoreDNS does not implement Store (wrong type for List method)
		have List() (map[string][]SrvRecord, error)
		want List() (map[string]SrvRecord, error)
make: *** [build] Error 2

```

--- a/cmd/common-main.go
+++ b/cmd/common-main.go
@@ -223,7 +223,11 @@ func handleCommonEnvVars() {
 	} else {
 		// Add found interfaces IP address to global domain IPS,
 		// loopback addresses will be naturally dropped.
-		updateDomainIPs(mustGetLocalIP4())
+		domainIPs := mustGetLocalIP4()
+		for _, host := range globalEndpoints.Hostnames() {
+			domainIPs.Add(host)
+		}
+		updateDomainIPs(domainIPs)
 	}
 
 	// In place update is true by default if the MINIO_UPDATE is not set

--- a/cmd/config/etcd/dns/etcd_dns.go
+++ b/cmd/config/etcd/dns/etcd_dns.go
@@ -48,8 +48,9 @@ func newCoreDNSMsg(ip string, port string, ttl uint32, t time.Time) ([]byte, err
 }
 
 // Close closes the internal etcd client and cannot be used further
-func (c *CoreDNS) Close() {
+func (c *CoreDNS) Close() error {
 	c.etcdClient.Close()
+	return nil
 }
 
 // List - Retrieves list of DNS entries for the domain.
@@ -259,7 +260,7 @@ func CoreDNSPath(prefix string) Option {
 }
 
 // NewCoreDNS - initialize a new coreDNS set/unset values.
-func NewCoreDNS(cfg clientv3.Config, setters ...Option) (*CoreDNS, error) {
+func NewCoreDNS(cfg clientv3.Config, setters ...Option) (Store, error) {
 	etcdClient, err := clientv3.New(cfg)
 	if err != nil {
 		return nil, err

--- a/cmd/config/etcd/dns/store.go
+++ b/cmd/config/etcd/dns/store.go
@@ -1,0 +1,27 @@
+/*
+ * MinIO Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dns
+
+// Store dns record store
+type Store interface {
+	Put(bucket string) error
+	Get(bucket string) ([]SrvRecord, error)
+	Delete(bucket string) error
+	List() (map[string][]SrvRecord, error)
+	DeleteRecord(record SrvRecord) error
+	Close() error
+}

--- a/cmd/data-update-tracker.go
+++ b/cmd/data-update-tracker.go
@@ -206,7 +206,7 @@ func (d *dataUpdateTracker) load(ctx context.Context, drives ...string) {
 			continue
 		}
 		err = d.deserialize(f, d.Saved)
-		if err != nil && err != io.EOF {
+		if err != nil && err != io.EOF && err != io.ErrUnexpectedEOF {
 			logger.LogIf(ctx, err)
 		}
 		f.Close()

--- a/cmd/endpoint.go
+++ b/cmd/endpoint.go
@@ -239,15 +239,15 @@ func (l EndpointZones) NEndpoints() (count int) {
 	return count
 }
 
-// Hosts - returns list of unique hosts
-func (l EndpointZones) Hosts() []string {
+// Hostnames - returns list of unique hostnames
+func (l EndpointZones) Hostnames() []string {
 	foundSet := set.NewStringSet()
 	for _, ep := range l {
 		for _, endpoint := range ep.Endpoints {
-			if foundSet.Contains(endpoint.Host) {
+			if foundSet.Contains(endpoint.Hostname()) {
 				continue
 			}
-			foundSet.Add(endpoint.Host)
+			foundSet.Add(endpoint.Hostname())
 		}
 	}
 	return foundSet.ToSlice()

--- a/cmd/globals.go
+++ b/cmd/globals.go
@@ -234,7 +234,7 @@ var (
 	globalBucketFederation bool
 
 	// Allocated DNS config wrapper over etcd client.
-	globalDNSConfig *dns.CoreDNS
+	globalDNSConfig dns.Store
 
 	// GlobalKMS initialized KMS configuration
 	GlobalKMS crypto.KMS

--- a/cmd/handler-api.go
+++ b/cmd/handler-api.go
@@ -44,8 +44,8 @@ func (t *apiConfig) init(cfg api.Config) {
 	}
 
 	apiRequestsMax := cfg.APIRequestsMax
-	if len(globalEndpoints.Hosts()) > 0 {
-		apiRequestsMax /= len(globalEndpoints.Hosts())
+	if len(globalEndpoints.Hostnames()) > 0 {
+		apiRequestsMax /= len(globalEndpoints.Hostnames())
 	}
 
 	t.requestsPool = make(chan struct{}, apiRequestsMax)

